charts/fleet/templates/deployment.yaml

```diff
@@ -224,11 +226,13 @@ spec:
       {{- end }}
 
       serviceAccountName: fleet-controller
-      nodeSelector: {{ include "linux-node-selector" . | nindent 8 }}
-{{- if $.Values.nodeSelector }}
```

Comment:
We seem to be omitting this now: if a node selector is present in the values, it will not be reflected in the deployment anymore :thinking: